packages/adapters-library/src/core/provider/CustomMulticallJsonRpcProvider.ts

```diff
@@ -39,8 +41,8 @@ export class CustomMulticallJsonRpcProvider extends CustomJsonRpcProvider {
     })
     this.multicallQueue = new MulticallQueue({
       fetchRequest,
-      maxBatchSize: 100,
-      flushTimeoutMs: 1000,
+      maxBatchSize,
+      flushTimeoutMs: 100,
```

Comment:
Reduced flush to 0.1s, which is what we had before. We shouldn't have to wait a whole second before new calls are received as that could delay responses by a lot.

Reply:
I think I accidentally change this on my last MR